DQM/HcalCommon/src/DQClient.cc

```diff
@@ -87,8 +87,16 @@ namespace hcaldqm
 			}
 		}
 
-		//	get the Channel Quality masks
+		// Initialize channel quality masks, but do not load (changed for 10_4_X, moving to LS granularity)
 		_xQuality.initialize(hashfunctions::fDChannel);
+	}
+	
+	/* virtual */ void DQClient::beginLuminosityBlock(DQMStore::IBooker&,
```

Comment:
@DryRun sorry why /* virtual */ ? If this is not needed please just remove it

Reply:
Hi @fabiocos  - I have no idea, actually... this is from before my time. I'll remove it.